maps/aurora/aurora-1_centcomm.dmm

```diff
@@ -8133,13 +8133,20 @@
 	},
 /area/centcom/living)
 "atb" = (
-/obj/machinery/vending/snack,
+/obj/machinery/vending/snack{
+	name = "Free Chocolate Corp";
```

Comment:
This (and all others in this PR) should be lowercase. This is an improper noun, not proper.

Reply:
...actually, I'm not sure if this is even improper.

Reply:
This is a proper noun. "Free Chocolate Corp" is the name of the corporation. Ergo, a proper noun.